src/App.js

```diff
@@ -1,11 +1,22 @@
 import React from 'react';
+import "./index.css"
 import Home from './pages/Home';
+import {Team} from "./pages/Team"
+import {BrowserRouter as Router,Link,Routes,Route} from "react-router-dom"
+
 
 const App = () => {
   return (
     
-      
-    <Home />
+    <div className='App'>
+      <Router>
+        <Routes>
+          <Route path="/" element={<Home/>}/>
+          <Route path="/teams" element={<Team/>}/>
+        </Routes>
+      </Router>
```

Comment:
Enclose the Router tag in the BrowserRouter tag, this will allow all the routes to be accessible in components rendered inside the BrowserRouter.
```suggestion
    <div className='App'>
    <BrowserRouter>
    <Router>
        <Routes>
          <Route path="/" element={<Home/>}/>
          <Route path="/teams" element={<Team/>}/>
        </Routes>
      </Router>
      </BrowserRouter>
```